generate_prometheus_targets.sh

```diff
@@ -45,7 +45,8 @@ for project in mlab-sandbox mlab-staging mlab-oti ; do
     # snmp_exporter on port 9116
     ./mlabconfig.py --format=prom-targets-sites \
         --template_target=s1.{{sitename}}.measurement-lab.org:9116 \
-        --label service=snmp_exporter > \
+        --label service=snmp \
+        --label __exporter-project=${project#mlab-} > \
```

Comment:
Please let's use just underscores in label names, i.e. `__exporter_project`.

Reply:
Fixed.